earth_enterprise/BUILD_RHEL_Centos.md

```diff
@@ -0,0 +1,121 @@
+# GEE Build Setup for CentOS, RHEL 6 and RHEL 7     
+
+
+## Install git  
+    
+Install the system default version:
+    
+```
+sudo yum install git
+```
+    
+Or install the latest version from the IUS repo (recommended) [ [More Info] ](https://ius.io/GettingStarted/):
+```
+sudo yum install -y wget
```

Comment:
I think you have to install epel_release before installing the ius repo.

Reply:
Moving the git section to *after* the additional repos.  Good catch!